environment.yml

```diff
@@ -9,6 +9,8 @@ dependencies:
   - pandas=0.25.1
   - scipy=1.3.1
   - tqdm=4.36
+  - matplotlib=3.1.2
+  - scikit-image=0.16.2
```

Comment:
To make the update working I had to add `- conda-forge` to `channels`

Reply:
This seems to be an issue with a local cache on my machine. Installing the environment on a different I also had to add the `conda-forge` channel.